google-cloud-logging/src/main/java/com/google/cloud/logging/LogEntryIterator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.logging;
+
+import com.google.common.collect.Lists;
+import com.google.logging.v2.TailLogEntriesResponse;
+import java.util.Iterator;
+import java.util.LinkedList;
+
+public class LogEntryIterator implements Iterator<LogEntry> {
```

Comment:
Document this method. While at first name explains it, but then you look at TailLogEntriesResponse object and without prior context it doesn't look obvious why.